theano/sandbox/gpuarray/blas.py

```diff
@@ -262,19 +284,14 @@ def c_code(self, node, name, inputs, outputs, sub):
         return code
 
     def c_code_cache_version(self):
-        return (1,)
+        return (3,)
 
     def c_headers(self):
         ret = super(GpuDot22, self).c_headers()
-        return ret + ['<numpy_compat.h>']
+        return ret + ['<numpy_compat.h>', '"gpuarray_helper.h"']
```

Comment:
Now this change isn't needed anymore. Also, numpy_compat is now added by blas op, so it isn't useful now, probably old stuff. So you can remove this method completly I think.